services/frontend-v3/src/components/userTable/UserTable.jsx

```diff
@@ -0,0 +1,420 @@
+import React, { Component } from "react";
+import UserTableView from "./UserTableView";
+import { LinkOutlined, SearchOutlined } from "@ant-design/icons";
+import { Select, Button, Input, Skeleton } from "antd";
```

Comment:
There shouldn't be antd in controllers

Reply:
Fixed, only have Skeleton for loading data.